### ISSUE TITLE
Does not reflect correctly when camera FOV changes from default value

### ISSUE THREAD

**Issue by JalalJaberi** [keijiro/AdamPlaneReflection#3]
If main camera be physical and sensorSize or focalLength changes, same as change in FOV value in non physical camera, the reflection position is not true anymore

**Comment by keijiro** [comment]
I'm not maintaining this project any more. The planer reflection component in HDRP does the same thing in a more sophisticated way. I'll keep this issue ticket opened for record purpose, please don't expect that this will be fixed.

**Comment by chaosmonger** [comment]
Bumping this issue... I know you are not maintaining this project anymore, but it's too complicate for me to move now to HDRP (need to change all my shaders/materials... and I've tons of custom ones), so I'm looking a solution like your great Plane Reflection, that, however, has the FOV problem...

**Comment by chaosmonger** [comment]
p.s. I've solved deleting the ProjectMatrix line.

**Comment by feshcruz** [comment]
Yeah It works if u delete the Project matrix line.... Anyway how should I give credits to it on my game? during the credits? 
thanks
